app/src/main/java/com/nervousfish/nervousfish/data_objects/tap/ATapData.java

```diff
@@ -20,9 +22,12 @@ public ATapData() {
     /**
      * <b>Should only be used for testing purposes!</b>
      *
-     * @param timestamp The timestamp of the tap data
+     * @param timestamp The timestamp of the tap data (NonNull)
```

Comment:
Can you remove this and add: `@throws IllegalArgumentException If the provided {@code timestamp} is {@code null}`